data/language/pl-PL.yml

```diff
@@ -1809,22 +1795,22 @@ strings:
   1765: '{COLOUR BLACK}Wartość Firmy'
   1766: '{COLOUR BLACK}Wartość Firmy ▼'
   1767: "{NEWLINE 1 2}{SPRITE}{NEWLINE 27 8}{STRINGID}"
-  1768: "{NEWLINE 0 3}{INT16_1DP}%        {NEWLINE 0 13}“{STRINGID}”"
-  1769: "{NEWLINE 0 3}{INT16_1DP}%{SPRITE 2325}{NEWLINE 0 13}“{STRINGID}”"
-  1770: "{NEWLINE 0 3}{INT16_1DP}%{SPRITE 2324}{NEWLINE 0 13}“{STRINGID}”"
+  1768: "{NEWLINE 0 3}{INT16_1DP}%{NEWLINE 0 13}{STRINGID}"
```

Comment:
Please leave the quotation marks here, rather than hardcoding them in the argument strings.